modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -979,9 +1064,17 @@ private CompletableFuture<Object> processSingleEntryAction(ReadWriteSingleRowRep
                 return lockFut.thenCompose(lockedRowId -> {
                     boolean inserted = lockedRowId == null;
 
-                    CompletableFuture raftFut =
+                    CompletableFuture<Object> raftFut =
                             lockedRowId == null ? applyCmdWithExceptionHandling(
-                                    new UpdateCommand(commitPartitionId, new RowId(partId), searchRow, txId)) :
+                                    msgFactory.updateCommand()
+                                            .tablePartitionId(msgFactory.tablePartitionIdMessage()
+                                                    .tableId(commitPartitionId.getTableId())
+                                                    .partitionId(commitPartitionId.getPartId())
+                                                    .build())
+                                            .rowUuid(Timestamp.nextVersion().toUuid())
```

Comment:
I think we should extract `Timestamp.nextVersion().toUuid()` into a method, like `generateNewRowId()` or something....

Reply:
I reviewed the code and it seems that all these occurrences are gone after latest main merged, so I'm ignoring this comment.